Source/Core/Core/HW/DSPHLE/MailHandler.cpp

```diff
@@ -95,9 +118,10 @@ void CMailHandler::DoState(PointerWrap &p)
 		// Restore queue.
 		for (int i = 0; i < sz; i++)
 		{
-			u32 value = temp.front();
+			u32 value = temp.front().first;
+			bool interrupt = temp.front().second;
 			temp.pop();
-			m_Mails.push(value);
+			m_Mails.push(std::make_pair(value, interrupt));
```

Comment:
emplace can also be used here and with the other two queue pushes above this in the diff.